examples/function_memory_game/README.md

```diff
@@ -0,0 +1,15 @@
+# rust-memory-game
+
+
+A port version of [react-memory-game](https://github.com/leftstick/react-memory-game). You are able to try it at [here](https://img.shields.io/website?label=demo&url=https%3A%2F%2Fexamples.yew.rs%2Ffunction_memory_game)](https://examples.yew.rs/function_memory_game)
```

Comment:
Can we adjust this to be more in line with the other examples? I.e. title, example badge.

Also that \[here] markdown is all over the place.

Reply:
WIll update it